src/components/templates/SinopiaResourceTemplates.jsx

```diff
@@ -8,12 +8,29 @@ import BootstrapTable from 'react-bootstrap-table-next'
 import Download from 'components/templates/Download'
 import { connect } from 'react-redux'
 import { newResource as newResourceCreator } from 'actionCreators/resources'
+import { rootResource } from 'selectors/resourceSelectors'
 
 /**
  * This is the list view of all the templates
  */
 class SinopiaResourceTemplates extends Component {
-  handleClick = resourceTemplateId => this.props.newResource(resourceTemplateId)
+  constructor(props) {
+    super(props)
+    this.state = { navigateEditor: false }
+  }
+
+  componentWillUpdate() {
+    // Forces a wait until the root resource has been set in state
+    if (this.state.navigateEditor && this.props.rootResource) {
+      this.props.history.push('/editor')
+    }
+  }
+
+  handleClick = (resourceTemplateId, event) => {
+    event.preventDefault()
```

Comment:
is the purpose of this line obvious to the better informed?  I am still struggling with it ...

Reply:
> is the purpose of this line obvious to the better informed? I am still struggling with it ...

The ` event.preventDefault` function is called here because the `Link` component is eventually rendered in the DOM as a button HTML element that has the default behavior of submitting the HTML form.

Reply:
a ha!  Thanks @jermnelson !